### PR TITLE
[TASK] Remove Extbase validation examples for TYPO3 v11

### PR DESCRIPTION
**Pull request by linawolf**
releases: main

### CODE REVIEW

File: Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_CustomValidator/_ObjectValidatorUsage.php [hunk 1]
```diff
@@ -18,9 +19,11 @@ class BlogController extends ActionController
      * $blog is a not yet persisted clone of the original blog containing
      * the modifications
      *
-     * @Validate(param="blog", validator="FriendsOfTYPO3\BlogExample\Domain\Validator\BlogValidator")
      * @throws NoBlogAdminAccessException
      */
+    #[Validate([
```
Comment (froemken) [inline Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_CustomValidator/_ObjectValidatorUsage.php:24]: IMO the method argument name is missing here. Else the validator will not know which argument it should validate.

**Review by froemken — CHANGES_REQUESTED** [review]

File: Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_CustomValidator/_ObjectValidatorUsage.php (hunk 1 above)
Comment (linawolf) [reply Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_CustomValidator/_ObjectValidatorUsage.php:24]: Do you know how to pass that?

File: Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_CustomValidator/_ObjectValidatorUsage.php [hunk 2]
```diff
@@ -18,9 +19,11 @@ class BlogController extends ActionController
...
+        'validator' => BlogValidator::class,
+    ])]
```
Comment (froemken) [inline Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_CustomValidator/_ObjectValidatorUsage.php:26]: Found some solutions in DummyController of TYPO3 test sources:

```suggestion
    #[Validate([
        'param' => 'blog',
        'validator' => BlogValidator::class,
    ])]
```

File: Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_CustomValidator/_ObjectValidatorUsage.php (hunk 1 above)
Comment (froemken) [reply Documentation/ExtensionArchitecture/Extbase/Reference/Domain/_CustomValidator/_ObjectValidatorUsage.php:24]: I have found a solution in TYPO3 test sources

**Comment by linawolf** [comment]
@froemken what else is needed to merge this?

**Review by froemken — APPROVED** [review]

**Pull request merged**